Nautilus/Documentation/tutorials/vehicle-module.md

```diff
@@ -0,0 +1,137 @@
+# Using the Vehicle Upgrade Module Gadget
+
+Vehicles modules are often hard to mod, and when multiple mods patch a function of a vehicle it can make severe conflicts leading to one of both mods to not work.  
+The Vehicle Upgrade Module Gadget is made to make your life of modder as easy as possible when it comes to vehicle modules.  
+
+This brand new gadget was made to be compatible with both Below Zero and Subnautica 1, some functions differ between Nautilus for SN1 and Nautilus for BZ.
+
+## Possible actions on completion
```

Comment:
Might want to rename the title here, to something like "Possibilities of custom vehicle upgrade modules"... idk, I don't have any great ideas

Also by any chance did you use the story goals guide as a basis?

Reply:
Yes I did take the story goal as a basis since I did not see anything like this on the site